### ISSUE TITLE
Support using the container standard output as input for the LogProxyConsumer

### ISSUE THREAD

**Issue by arturo-seijas** [canonical/loki-k8s-operator#185]
### Enhancement Proposal

Currently, log filenames need to be specified for the `LogProxyConsumer` to be able to scrap them. It would be nice to support gathering the logs from the containers' standard output, which would be beneficial in several ways:

- No need to manage rotation of the logs files
- Reduced risk of losing entries in case a container stops between the write and the scraping of the file
- Logs could be queried without the logging infrastructure without additional tooling, just by using kubectl
- It's the de facto standard

**Comment by jnsgruk** [comment]
This will likely be very easy for us to support once pebble lands its new log forwarding feature :)

**Comment by simskij** [comment]
This is exactly what the Pebble Log Forwarding feature aims to solve. Closing this in favor of https://github.com/canonical/pebble/pull/256
